### DISCUSSION TITLE
Add a command to select the version

### DISCUSSION THREAD

**Discussion by alex-way** [biomejs/biome-vscode#210] [Ideas]
It's seemingly not possible to select a version after one has already been selected.

It would be great if you could run a command to select a new version to use.

**Comment by ematipico** [comment]
It already exists. Have you tried the command palette?

**Comment by alex-way** [comment]
<img width="608" alt="Screenshot 2024-05-02 at 19 32 34" src="https://github.com/biomejs/biome-vscode/assets/29122169/a4b3caac-528a-40c0-983a-9371bd141e7d">

I'm not able to see any option available for that?

The extension version I have is `v2.2.2`

**Reply by ematipico** [reply]
This is only possible when you used the **bundled** binary (hence you don't have Biome installed via npm).

If you click on the bottom right the "Biome v* (bundled)" bar, you'll be able to choose the version

**Comment by alex-way** [comment]
I'm also not able to see a registered command under https://github.com/biomejs/biome-vscode/blob/main/src/main.ts unless I'm also mistaken
